packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -61,7 +62,9 @@ export class WorkspaceCommandContribution implements CommandContribution {
         @inject(FileSystem) protected readonly fileSystem: FileSystem,
         @inject(WorkspaceServer) protected readonly workspaceServer: WorkspaceServer,
         @inject(SelectionService) protected readonly selectionService: SelectionService,
-        @inject(OpenerService) protected readonly openerService: OpenerService
+        @inject(OpenerService) protected readonly openerService: OpenerService,
+        @inject(EditorManagerImpl) private editorService: EditorManagerImpl
```

Comment:
program against the interface not implementation, i.e. `EditorManager`

Reply:
I moved the code over to the EditorManager, this introduced a dependency on the editor package. But now since the editor package also has a dependency on workspace, it won't compile, complaining that editor package doesn't exist (circular dependency). Any idea ?

Reply:
Could you try to remove a dependency from `@theia/editor` to `@theia/workspace`? The latter does not seem to be used in the former.